configs/openfin/manifest-local.json

```diff
@@ -58,8 +58,8 @@
     "fileName": "ChartIQ-local-installer",
     "appAssets": [
         {
-            "src": "https://files.chartiq.com/finsemble-dev/Assimilation_3_11_0.zip",
-            "version": "3.11.0_48",
+            "src": "https://assets.finsemble.com/assimilation/dev/assimilation_pr50.zip",
```

Comment:
This looks like we are referencing a PR in a manifest file, which seems strange.  But I don't know the context.  Could you explain why this change is necessary?

Reply:
Its because it was checked out from planned/3.11.0 instead of the newer  branches.